.github/workflows/test.yml

```diff
@@ -53,6 +62,15 @@ jobs:
           rails: '6.1'
           gemfile: gemfiles/rails-6.1.gemfile
           experimental: false
+        - mongodb: '4.4'
+          ruby: ruby-3.1
+          topology: replica_set
+          task: test
+          os: ubuntu-20.04
```

Comment:
Is Ubuntu 22.04 supported?

Reply:
let's give it a try

Reply:
Well, it doesn't work, even Mongo team [runs on Ubuntu 20.04](https://github.com/mongodb/mongoid/blob/master/.github/workflows/test.yml#L24), so, let's stick with this.